.github/dependabot.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  ignore:
+  - dependency-name: view_component
+    versions:
+    - 2.26.0
+    - 2.31.0
+  - dependency-name: rails
+    versions:
+    - 6.1.1
+    - 6.1.2
+    - 6.1.2.1
+    - 6.1.3
```

Comment:
@tillprochaska do you know maybe why so many dependencies are ignored? If not, I would probaby remove that section and see what happens.

Reply:
When you close Dependabot PRs, Dependabot remembers that so it doesn’t open a PR for the same version update the next time it runs. I guess that’s their way to port over that state to the native version of Dependabot.

I think it’s fine to remove those ignored version though! :)